edgelet/build/windows/package.ps1

```diff
@@ -1,8 +1,17 @@
-Param([Switch] $CreateTemplate, [Switch] $CreateCab, [Switch] $SkipInstallCerts)
+Param(

```

Comment:
File has a mix of line endings. Please normalize to CRLF.

Reply:
fixed